.flake8

```diff
@@ -0,0 +1,11 @@
+[flake8]
```

Comment:
I know this is merged, we typically use `pflake8` so that we can include the `flake8` configuration in the `pyproject.toml` file: https://pypi.org/project/pyproject-flake8/

Reply:
I saw this file in other charms so I thought that would be necessary.

Reply:
Yeh I have seen it a few times as well and didn't comment, I don't think it is necessary. Maybe try to remove it and see if the `flake8` commands still succeed?

Reply:
I did a test and if this file is present, is considered instead of pyproject.toml.
But since they are different, what would be the right one to be used?

Pyproject:
```
select = ["E", "W", "F", "C", "N", "R", "D", "H"]
```

.flake8:
```
select: E,W,F,C,N
```

Reply:
The most restrictive one working, I guess

Reply:
https://github.com/canonical/content-cache-k8s-operator/pull/10